### PR TITLE
[zh-cn] sync kubeadm_init_phase_kubelet-start

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-start.md

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-start.md [hunk 1]
```diff
@@ -104,6 +104,20 @@ kubelet-start 操作的帮助命令
...
+<tr>
+<td colspan="2">--image-repository string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "registry.k8s.io"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Choose a container registry to pull control plane images from
+-->
+选择一个容器镜像仓库来从中提取控制平面组件的镜像
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-start.md:116]:
```suggestion
选择一个容器镜像仓库来从中拉取控制平面组件的镜像
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-start.md [hunk 2]
```diff
@@ -104,6 +104,20 @@ kubelet-start 操作的帮助命令
 </td>
 </tr>
 
+<tr>
+<td colspan="2">--image-repository string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "registry.k8s.io"</td>
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-start.md:108]: `Default:` can be translated too

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**